clients/java/src/test/java/marquez/client/models/StreamTest.java

```diff
@@ -17,16 +17,23 @@
 import static marquez.client.models.ModelGenerator.newStream;
 import static org.assertj.core.api.Assertions.assertThat;
 
+import com.fasterxml.jackson.core.JsonProcessingException;
+import java.util.UUID;
+import marquez.client.Utils;
 import org.junit.jupiter.api.Test;
 
 @org.junit.jupiter.api.Tag("UnitTests")
 public class StreamTest {
-  private static final Dataset STREAM = newStream();
```

Comment:
I would add a helper method `newStream(UUID)`, where `UUID` is the current version you'd like to assert. In the test below, I would still us `Stream.fromJson(JSON)` and not expose the mapper.

Reply:
`newStream()` method is used only in this test, so I'll just change method signature for it.
You are right `Stream.fromJson(JSON)` is better approach.